lib/app.rb

```diff
@@ -50,9 +49,10 @@ class App < Sinatra::Base
     use Routes::Comments
     use Routes::Nits
     use Routes::Teams
+    use Routes::Sessions
```

Comment:
Turns out that once Teams is fixed with please_login it doesn't matter where Sessions is.